kasa/aestransport.py

```diff
@@ -184,8 +185,24 @@ async def send_secure_passthrough(self, request: str) -> Dict[str, Any]:
             assert self._encryption_session is not None
 
         raw_response: str = resp_dict["result"]["response"]
-        response = self._encryption_session.decrypt(raw_response.encode())
-        return json_loads(response)  # type: ignore[return-value]
+
+        try:
+            response = self._encryption_session.decrypt(raw_response.encode())
+            ret_val = json_loads(response)
+        except Exception as ex:
+            try:
+                ret_val = json_loads(raw_response)
+                _LOGGER.debug(
+                    "%s Secure passthrough response was received unencrypted!",
```

Comment:
```suggestion
                    "Received unencrypted response over secure passthrough from %s",
```
Although I'm not sure if it's worth logging, rather than adding a comment why this is done?

Reply:
I've committed the suggested change but I think for the time being it's useful to log it so we can see if it's happening at other times we weren't expecting on devices that may be experiencing issues.